alpakka-zeromq/src/main/java/no/sysco/middleware/alpakka/zeromq/javadsl/Zmq.java

```diff
@@ -6,48 +6,48 @@
 import akka.stream.javadsl.Flow;
 import akka.stream.javadsl.Sink;
 import akka.stream.javadsl.Source;
-import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZeroMQPublishStage;
-import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZeroMQPullStage;
-import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZeroMQPushStage;
-import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZeroMQSubscribeStage;
+import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZmqPublishStage;
+import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZmqPullStage;
+import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZmqPushStage;
+import no.sysco.middleware.alpakka.zeromq.javadsl.internal.ZmqSubscribeStage;
 import org.zeromq.ZMsg;
 
-public class ZeroMQ {
+public class Zmq {
```

Comment:
I would suggest adding javadoc to explain what different methods do (might be obvious to those who understand ZeroMQ API?). e.g. explaining the difference between pullServerSource and pullClientSource.